File: Prod.RutaDigital.Entidades/Premio/PremioResponse.cs
namespace Prod.RutaDigital.Entidades;

public class PremioResponse : Premio
{
    public string? nivel_madurez { get; set; }
    public string? tipo_canje { get; set; }    
    public byte[]? numArray { get; set; }
    
    public string? etiqueta { get; set; }
    public string? foto_tipo { get; set; }
    public byte[]? fotoTipoArray { get; set; }
}

File: Prod.RutaDigital.Presentacion.Angular/ClientApp/src/app/catalogo-premios/interfaces/premio.response.ts
import { SafeUrl } from '@angular/platform-browser';

export interface PremioResponse {
  id_premio?: number;
  id_tipo_premio?: number;
  tipo_premio?: string;
  id_nivel_madurez?: number;
  id_tipo_canje?: number;
  foto?: string;
  nombre?: string;
  descripcion_corta?: string;
  detalle_premio?: string;
  puntos_produce?: number;
  correo?: string;

  nivel_madurez?: string;
  tipo_canje?: string;
  numArray?: Blob;

  imagenPremio?: SafeUrl;

  etiqueta?: string;
  foto_tipo?: string;
  fotoTipoArray?: Blob;

  imagenTipoPremio?: SafeUrl;
}
